code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -215,33 +281,91 @@ func commitProcessed(ctx context.Context) {
 	}()
 
 	db := datastore.GetStore().GetDB()
-	var challenges []*ChallengeEntity
+	count := 0
 
-	db.Where(ChallengeEntity{Status: Processed}).
-		Order("sequence").
-		Find(&challenges)
+	rows, err := db.Model(&ChallengeEntity{}).
+		Where("status = ?", Processed).
+		Select("challenge_id", "created_at").Rows()
 
-	if len(challenges) > 0 {
+	if err != nil {
+		logging.Logger.Error("[challenge]validate: ",
```

Comment:
commit
shouldn't we be making logs better to tell where in commit also error happened - doing what process..

Reply:
zap will log code line for any error. set prefix with fixed `[challenge]commit` is easier to check it from logs

Reply:
fixed typo.